lib/msprime.c

```diff
@@ -1504,6 +1507,14 @@ msp_cut_right_break(msp_t *self, segment_t *lhs_tail, segment_t *y, segment_t *n
     y->next = NULL;
     y->right = track_end;
     fenwick_increment(&self->links[label], y->id, (int64_t) track_end - (int64_t) new_segment->right);
+    if (msp_has_breakpoint(self, (uint32_t) track_end)) {
```

Comment:
Remove redundant else here since we're not using the first clause.

Reply:
done